tests/kafkatest/tests/kafka_test.py

```diff
@@ -28,18 +28,20 @@ class KafkaTest(Test):
     setUp. The Zookeeper and Kafka services are available as the fields
     KafkaTest.zk and KafkaTest.kafka.
     """
-    def __init__(self, test_context, num_zk, num_brokers, topics=None):
+    def __init__(self, test_context, num_zk, num_brokers, topics=None, use_new_coordinator=None):
```

Comment:
`use_new_coordinator` is specified in the constructor instead of in the test matrix, or python will complain that the test name is too long (too many variables in the matrix).

Reply:
Based on my previous comment, I wonder if we need this change.